api/routes/tokens.py

```
@@ -0,0 +1,152 @@
+
+            logging.info("Token generated for user: %s", user_email)
+
+            return TokenListItem(
+                token_id=api_token,
+                created_at=111 # TODO set actual created_at timestamp
+            )
+
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="Failed to generate token"
+        )
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)